.lgtm.yml

```diff
@@ -0,0 +1,45 @@
+# LGTM config file docs: https://lgtm.com/help/lgtm/lgtm.yml-configuration-file
+
+# Results from files under any classifier will be excluded from LGTM stats.
+path_classifiers:
+  library:
+    - "auxil/broker/caf/"
```

Comment:
Just wondering, wouldn't it make sense to enable LGTM for Broker separately and excluding `auxil/broker` here?

Reply:
Good question.  Not totally sure due to lack of experience w/ LGTM but my understanding of a few points of how this will work:

* The config file (and classifier tags) won't actually enable/disable analysis.  E.g. Broker is already being analyzed separately (never even gave LGTM any specific instructions to do that): https://lgtm.com/projects/g/zeek/broker/

* A file that has a classifier tag is excluded from a project's overall "LGTM stats".  Guess that's things like defect density and code quality grading.

* A file that has a classifier tag causes alerts in it to be hidden by default, but someone can still explicitly query for alerts in them if they want.

Given those, main thing is wanting to exclude external/3rdparty code from being counted in stats.  Secondary thing is picking classifier tags that make the default view of alerts convenient.  Guessing it may be more convenient to have a wide view of alerts that includes sub-projects the Zeek-organization is directly responsible for, but can always filter down later.

Reply:
> may be more convenient to have a wide, centralized view of alerts that includes sub-projects the Zeek-organization

Yeah, makes sense to see it like that. 👍